src/pages/registration/contributors_tab/components/ContributorEditors.tsx

```diff
@@ -0,0 +1,59 @@
+import { ErrorMessage, FieldArray, FieldArrayRenderProps, useFormikContext } from 'formik';
+import React, { useEffect, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { FormHelperText, MuiThemeProvider, Typography } from '@material-ui/core';
+import BackgroundDiv from '../../../../components/BackgroundDiv';
+import lightTheme from '../../../../themes/lightTheme';
+import { ContributorRole } from '../../../../types/contributor.types';
+import { ContributorFieldNames } from '../../../../types/publicationFieldNames';
+import { Registration } from '../../../../types/registration.types';
+import Authors from '../Authors';
+
+export const ContributorEditors = () => {
+  const { t } = useTranslation('registration');
+  const {
+    values: {
+      entityDescription: { contributors },
+    },
+    errors,
+    touched,
+    setFieldValue,
+  } = useFormikContext<Registration>();
+  const contributorsError = errors.entityDescription?.contributors;
+  const contributorsTouched = touched.entityDescription?.contributors;
+  const editorsRef = useRef(contributors);
+
+  useEffect(() => {
+    const editors = contributors
+      .filter((contributor) => contributor.role === ContributorRole.EDITOR)
+      .map((contributor, index) => ({ ...contributor, sequence: index + 1 }));
+    editorsRef.current = editors;
+    return () => {
+      editorsRef.current = [];
```

Comment:
hvorfor vil vi nullstille på unmount? betyr ikke det at man mister alle editors man har lagt til om man bytter fane uten å lagre?

Reply:
Bare ref'en som nullstilles, ikke Formik-verdiene. Fikk React-feilmeldingen `To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function`

Reply:
hm, spennende. får den feilen i ny og ne nå også, så kan være det er samme greia flere plasser

Reply:
var kanskje pga den refen som ikke skulle være dependency. fikser